cmd/standalone/main.go

```diff
@@ -221,12 +221,16 @@ func startQuery(
 		queryApp.HandlerOptions.Prefix(qOpts.Prefix),
 		queryApp.HandlerOptions.Logger(logger),
 		queryApp.HandlerOptions.Tracer(tracer))
+	r := mux.NewRouter()
 	staticHandler, err := queryApp.NewStaticAssetsHandler(qOpts.StaticAssets, qOpts.UIConfig)
 	if err != nil {
 		logger.Fatal("Could not create static assets handler", zap.Error(err))
 	}
-	r := mux.NewRouter()
-	apiHandler.RegisterRoutes(r)
+	if staticHandler != nil {
+		apiHandler.RegisterRoutes(r)
```

Comment:
this is confusing. Better to group related code (and use scopes to hide vars that's not needed later)
```
r := mux.NewRouter()

{ 
    h := New...
    h.RegisterRoutes(r)
}

if h, err := queryApp.NewStaticAssetsHandler(qOpts.StaticAssets, qOpts.UIConfig); err != nil {
    logger.Fatal("Could not create static assets handler", zap.Error(err))
} else if h != nil {
    h.RegisterRoutes(r)
} else {
    logger.Info("Static handler is not registered")
}
```

Reply:
I have added a function which handles static handler registration